src/components/Configure/InstallIntegration.tsx

```diff
@@ -23,12 +23,34 @@ function useForceUpdate() {
   return { seed, reset };
 }
 
+export type FieldMappingsType = { [key: string]: Array<IntegrationFieldMapping> };
```

Comment:
@jatinsandilya Can you rename this to `FieldMappings`? We shouldn't post-pend "Type" at the end of types, it's unnecessarily verbose. Also you'll have to explicitly re-export it in https://github.com/amp-labs/react/blob/main/src/public/index.ts (and that file is exported in https://github.com/amp-labs/react/blob/main/src/index.ts). 

So then builders can directly use the type in their code like so:

```
import { FieldMappings, InstallIntegration } from '@amp-labs/react';
```